initcwnd.rb

```diff
@@ -56,12 +56,15 @@
 # The reason I didn't is because curl supports SSLKEYLOGFILE which is very
 # convenient for Wireshark.
 # TODO: We do not send --compressed
-site_html = `SSLKEYLOGFILE=key.log curl --tlsv1.2 -k -H "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36" --fail --http1.1 #{uri.to_s}`
+# TODO: Support TLS 1.3
+site_html = `SSLKEYLOGFILE=key.log curl --tlsv1.2 --tls-max 1.2 -k -H "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36" --fail --http1.1 #{uri.to_s}`
```

Comment:
Did you need `--tls-max` here because it doesn't work with the key log file for 1.3?

Reply:
Indeed: if curl was able to do TLS 1.3 the logic in the script wouldn't work and produce no useful output. 

I think having this strict here to show it works under certain known conditions and not under others will make it easier for other people to pick up the needed work :)